dot/config_integration_test.go

```diff
@@ -0,0 +1,30 @@
+// Copyright 2021 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+//go:build integration
+// +build integration
+
+package dot
+
+import (
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestExportConfig(t *testing.T) {
+	cfg, cfgFile := newTestConfigWithFile(t)
+	require.NotNil(t, cfg)
```

Comment:
Do we need these? Isn't this just testing our test code? Maybe you can batch remove these? 🤔

Reply:
Removed.